core/src/main/java/io/confluent/connect/storage/Storage.java

```diff
@@ -53,30 +56,34 @@
   boolean create(String path);
 
   /**
-   * Open for reading an object at the given path.
-   *
-   * @param path the path of the object to be read.
-   * @param conf storage configuration.
-   * @return a seek-able input stream associated with the requested object.
-   */
-  SeekableInput open(String path, C conf);
-
-  /**
    * Creates a new object in the given path and with the given configuration.
    *
-   * In stores supporting weak consistency models (e.g. eventual consistency) the success of this operation might depend on
-   * successfully completing all operations on the output stream. For instance successfully closing the stream after
-   * writing all the data. Additionally, the effect of this operation might not be immediately visible to subsequent
+   * <p>In stores supporting weak consistency models (e.g. eventual
+   * consistency) the success of this operation might depend on
+   * successfully completing all operations on the output stream.
+   * For instance successfully closing the stream after
+   * writing all the data. Additionally, the effect of this
+   * operation might not be immediately visible to subsequent
    * operations.
    *
    * @param path the path of the object to be created.
    * @param conf storage configuration.
-   * @param overwrite whether to override an existing object with the same path (optional operation).
+   * @param overwrite whether to override an existing object with
+   *                  the same path (optional operation).
```

Comment:
I now see that the checkstyle property that complains about alignment here is: `JavadocTagContinuationIndentation`
How strong do we feel about this property? Seems to create problems with intellij: https://github.com/checkstyle/checkstyle/issues/3258